ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,313 @@
+                            Text(title)
+                                .font(Font(tokens.textFont))
+                                .foregroundColor(Color(tokens.leadingTextColor))
+                                .lineLimit(cell.titleLineLimit)
+                        }
+                        if let subtitle = cell.subtitle {
+                            if subtitle != "" {
+                                Text(subtitle)
+                                    .font(Font(tokens.subtitleFont))
+                                    .foregroundColor(Color(tokens.subtitleColor))
+                                    .lineLimit(cell.titleLineLimit)
+                            }
+                        }
+                    }
+                    Spacer()
+                    HStack {
+                        if let trailingIcon = cell.trailingIcon {
+                            if trailingIcon != .none {
+                                let isDisclosure = trailingIcon == .disclosure
+                                let disclosureSize = tokens.disclosureSize
+                                let iconSize = tokens.iconSize
+                                Image(uiImage: trailingIcon.icon!)
+                                    .resizable()
+                                    .foregroundColor(Color(isDisclosure ? tokens.disclosureIconForegroundColor : tokens.trailingItemForegroundColor))
+                                    .frame(width: isDisclosure ? disclosureSize : iconSize,
+                                           height: isDisclosure ? disclosureSize : iconSize)
+                                    .padding(.leading, isDisclosure ? tokens.disclosureInterspace : tokens.iconInterspace)
+                            }
+                        }
+                    }
+                }
+            })
+            .buttonStyle(ListCellButton(tokens: tokens, layoutType: layoutType))
+        }
+    }
+
+    struct ListCellButton: ButtonStyle {
+        let tokens: MSFListTokens
+        let layoutType: MSFListCellVnextLayoutType
+        func makeBody(configuration: Self.Configuration) -> some View {
+            return configuration.label
+                .contentShape(Rectangle())
+                .frame(minHeight: layoutType.height)
+                .listRowInsets(EdgeInsets())
+                .padding(.leading, tokens.horizontalCellPadding)
+                .padding(.trailing, tokens.horizontalCellPadding)
+                .background(configuration.isPressed ? Color(tokens.highlightedBackgroundColor) : Color(tokens.backgroundColor))
+        }
+    }
+
+    /// View for List Header (availble for iOS 14.0+)
+    struct Header: View {
+        let title: String
+        var tokens: MSFListTokens
+
+        init(title: String, tokens: MSFListTokens) {
+            self.title = title
+            self.tokens = tokens
+        }
+
+        var body: some View {
+            GeometryReader { _ in
```

Comment:
Why do you need a GeometryReader here?

Reply:
Headers are kinda wonky right now where they're not meant to be very customizable. Adding GeometryReader forces the view to expand (kind of like a Spacer) so that the default background color does not show. Below is a screenshot of what it looks like with the GeometryReader removed. I've also tried resetting list row insets, padding, frame and the header view did not seem to budge.

![Screen Shot 2020-12-30 at 5 52 30 PM](https://user-images.githubusercontent.com/22566866/103387139-cd67a080-4ac7-11eb-96e8-e93b3069798a.png)